ai_guide/clients/aws_polly_client.py

```diff
@@ -0,0 +1,42 @@
+import os
+from contextlib import closing
+
+import boto3
+from dotenv import load_dotenv
+
+from ai_guide.settings import MEDIA_ROOT
+
+from .config import OUTPUT_FORMAT, REGION_NAME, VOICE_ID
+
+load_dotenv()
+AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID')
+AWS_SECRET_ACCESS_KEY = os.getenv('AWS_SECRET_ACCESS_KEY')
+
+
+class AwsPollyClient:
+
+    def get_audio(self, text: str, file_name: str):
+        try:
+            polly_client = boto3.Session(
```

Comment:
I'm not sure here.
Is the session should be created every time we make a request to Polly?